omigami/tests/test_spectra_matching.py

```diff
@@ -0,0 +1,148 @@
+from io import StringIO
```

Comment:
I did it in many different ways but I couldn't make this diff contain only the changes I made in the file. This file was renamed (it was test_endpoint.py) and I added the **last 4 tests**